osu.Game.Tests/Visual/UserInterface/ThemeComparisonTestScene.cs

```diff
@@ -14,18 +14,18 @@ namespace osu.Game.Tests.Visual.UserInterface
 {
     public abstract partial class ThemeComparisonTestScene : OsuGridTestScene
     {
-        private readonly bool showNoColourProvider;
+        private readonly bool showWithoutColourProvider;
```

Comment:
Oh haha, this is the name I chose initially for the variable. It read slightly weird to me so I replaced `Without` with `No`.